packages/flutter/lib/src/cupertino/split_form_row.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'theme.dart';
+
+export 'split_form_row.dart';
+
+/// Creates a standard split row, with a helper text to one side and a child
+/// widget to the other.
+///
+/// A [leadingText] is required. This string is displayed in a [Text] widget
+/// leading to the side of the [CupertinoSplitFormRow]. It can be set to
+/// an empty string, although it is encouraged to set the [leadingText] in
+/// order to follow standard iOS practices.
+///
+/// {@tool snippet}
+///
+/// Creates a [CupertinoFormSection] containing a [CupertinoSplitFormRow] with a
+/// [leadingText], [child] widget, [helperText] and [errorText].
+///
+/// ```dart
+/// class FlutterDemo extends StatefulWidget {
+///   FlutterDemo({Key key}) : super(key: key);
+///
+///   @override
+///   _FlutterDemoState createState() => _FlutterDemoState();
+/// }
+///
+/// class _FlutterDemoState extends State<FlutterDemo> {
+///   bool toggleValue = false;
+///
+///   @override
+///   void initState() {
+///     super.initState();
+///   }
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return CupertinoPageScaffold(
+///       child: Center(
+///         child: CupertinoFormSection(
+///           title: "Section 1",
+///           children: [
+///             CupertinoFormSplitRow(
+///               text: 'Toggle',
+///               child: CupertinoSwitch(
+///                 value: this.toggleValue,
+///                 onChanged: (value) {
+///                   setState(() {
+///                     this.toggleValue = value;
+///                   });
+///                 },
+///               ),
+///             ),
+///           ],
+///         ),
+///       ),
+///     );
+///   }
+/// }
+/// ```
+/// {@end-tool}
+class CupertinoSplitFormRow extends StatefulWidget {
```

Comment:
Optional: can we just call this CupertinoFormRow? 1- to make it a bit more consistent with CupertinoTextFormFieldRow (or is it CupertinoTextFieldFormRow?). 2- I think the hierarchy might be a bit more glanceable. That a CupertinoFormRow is in a CupertinoFormSection. CupertinoSplitFormRow sounds more specialized and force people to take a double take on which goes into what.